dangerfile.js

```diff
@@ -306,6 +315,25 @@ const tasks = [
     // }
 ];
 
+const runTasks = async tasks => {
+    const errors = [];
+    for (const task of tasks) {
+        try {
+            await task();
+        } catch (e) {
+            errors.push({ task: task.name, error: e.stdout || e });
+        }
+    }
+    return errors;
+};
```

Comment:
It's not like performance in CI is *super* important, but could these tasks be run in parallel?

Reply:
I think that if Danger is going to be used more and expanded then this file should be refactored in a number of ways. I think that should be a separate endeavor outside of this pr though. Mostly what I am doing here is capturing all the errors at once and reporting all the errors, rather than only reporting the first error hit.